src/kirby/scss/_global-styles.scss

```
@@ -8,6 +8,9 @@
   @each $color, $color-name in $kirby-colors {
     --kirby-#{$color}: #{$color-name};
   }
+  @each $color, $color-name in $text-colors {
+    --kirby-text-#{$color}: #{$color-name};
```

Include `color` in variable name:
```suggestion
    --kirby-text-color-#{$color}: #{$color-name};
```